lib/matplotlib/__init__.py

```diff
@@ -207,6 +207,36 @@ def _check_versions():
     sys.argv = ['modpython']
 
 
+def set_verbose(level='debug'):
+    """
+    Shortcut to set the debugging level of the logging module.
+
+    Parameters
+    ----------
+    level : str or bool
+        If True set logging level to "DEBUG" for matplotlib module.
+        If FALSE set logging level to "WARNING" for matplotlib module (this
+        is the default level if ``verbose`` is not called). If a string, must
+        be one of ['warning', 'info', 'debug'], in order of increasing
```

Comment:
But standard Python logging has more levels than this?

Reply:
Yeah there is critical and something else I can’t remember.  Did we want to support quieting the more critical levels?   To my knowledge we never use those levels anyhow.